static/js/unread_ops.js

```diff
@@ -185,14 +185,20 @@ export function process_unread_messages_event({message_ids, message_details}) {
     }
 
     for (const message_id of message_ids) {
+        const message_info = message_details[message_id];
+        if (!message_info) {
+            // Server sent us the `message_id` but didn't sent us the `message_details` for
+            // it. This case is easy to reproduce after hitting an API limit when running `mark_as_unread_from_here`.
+            // TODO: This is most likely a server side bug and needs to be fixed there.
+            continue;
+        }
+
         const message = message_store.get(message_id);
 
         if (message) {
             message.unread = true;
         }
 
-        const message_info = message_details[message_id];
-
         let user_ids_string;
```

Comment:
Oh! We actually want to process the `message_id` here, regardless of if the server sent us the `message_details` for it.